test/reactionsystem_core/coupled_equation_crn_systems.jl

```diff
@@ -972,7 +975,8 @@ end
 # Checks that various erroneous coupled system declarations yield errors.
 let
     @parameters p1 p2
-    @variables τ  U1(τ) V1(t)
```

Comment:
This causes a a `@test_throws` to go from throwing an error to not throwing an error (hence causing test failure). The reason is not directly obvious, but should be possible to figure out since I've been able to look closer at it.

Reply:
Interesting.